bedrock/mozorg/forms.py

```diff
@@ -58,13 +58,13 @@ def render(self, name, value, attrs=None):
         input_txt = super(PrivacyWidget, self).render(name, value, attrs)
 
         policy_txt = _(u'I’m okay with Mozilla handling my info as explained '
-                       u'in <a href="%s">this Privacy Policy</a>')
+                       u'in <a href="%s">this Privacy Notice</a>')
```

Comment:
This one would be pretty tough to do, but I think it's possible. There are some global variables in Django that tell us some l10n info that we could use. The other issue here is that we also don't know what template is being used, and therefore don't know which lang file to check for being active.